doc/classes/AudioEffectDistortion.xml

```diff
@@ -2,11 +2,11 @@
 <class name="AudioEffectDistortion" inherits="AudioEffect" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
 		Adds a distortion audio effect to an Audio bus.
-		Modify the sound to make it distorted.
+		Modifies the sound to make it distorted.
```

Comment:
I wonder how much this line actually adds, like what does it add that the above one doesn't?

Reply:
It's just a nitpick. If you look at the other audio effects, the descriptions are written as "_actions_ the _noun_", instead of "_action_ the _noun_", so I figured this could be consistent with that trend.